### PR TITLE
WE-304 Pull content from `main`

### PR DESCRIPTION
**Pull request by jonambas**
### Support Docs Update Checklist

Note: when you open a pull request, your changes are published to https://staging.sparkpost.com/docs.

Please take a moment to check each of these steps for your pull request:

- [ ] Give your pull request a meaningful name.
- [ ] Include a description of your changes in the pull request.
- [ ] _SparkPost staff only_: Check that your article appears [here](https://staging.sparkpost.com/docs) and looks correct.
- [ ] Check the links in your article.
- [ ] Use lowercase filenames.

### CODE REVIEW

File: content/momentum/3/index.md [hunk 1]
```diff
@@ -16,4 +18,4 @@ Confidential & Proprietary.
 
 This book documents Message Systems Momentum.
 
----
+* * *
```
Comment (EvanYoung-SP) [inline content/momentum/3/index.md:21]: This change is undoing some of the changes made here: https://github.com/SparkPost/support-docs/pull/612/files

File: content/momentum/4/index.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
 ---
-lastUpdated: '03/26/2020'
-title: 'Momentum 4.x'
-description: 'Message Systems Inc Copyright 2014 Message Systems Inc Confidential Proprietary Abstract This book documents Momentum 4 Document generated on 2018 Aug 28 Table of Contents Preface 1 Typographical Conventions Used in This Document I Introduction to Momentum 1 Components 2 Life of A Message 3 Roles and Behaviors 4 Licensed...'
+lastUpdated: "03/26/2020"
```
Comment (EvanYoung-SP) [inline content/momentum/4/index.md:2]: This change is undoing some of the changes made here: https://github.com/SparkPost/support-docs/pull/612/files

File: content/momentum/changelog/index.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+name: "Momentum Changelogs"
+description: "Momentum Changelogs"
 ---
 
-- [Momentum 4.x Changelogs](/momentum/changelog/4)
-- [Legacy Momentum Changelogs](/momentum/changelog/legacy)
-
----
+* [Momentum 4.x Changelogs](/momentum/changelog/4)
+* [Legacy Momentum Changelogs](/momentum/changelog/legacy)
```
Comment (EvanYoung-SP) [inline content/momentum/changelog/index.md:10]: This change is undoing some of the changes made here: https://github.com/SparkPost/support-docs/pull/612/files

File: content/momentum/mobile/index.md [hunk 4]
```diff
@@ -12,6 +14,4 @@ Confidential & Proprietary.
 
 **Abstract**
 
-This book is an administration guide for Mobile Momentum.
-
----
+This book is an administration guide for Mobile Momentum.
```
Comment (EvanYoung-SP) [inline content/momentum/mobile/index.md:17]: This change is undoing some of the changes made here: https://github.com/SparkPost/support-docs/pull/612/files

**Review by EvanYoung-SP — CHANGES_REQUESTED** [review]
A few changes will need to be made since they're undoing the intentional changes made in a previous PR. Not a big deal.
 https://github.com/SparkPost/support-docs/pull/612/files

**Comment by jonambas** [comment]
@EvanYoung-SP waiting for #614 to be merged before i finish up this one, since there are content changes in that PR

**Review by EvanYoung-SP — APPROVED** [review]
LGTM!

**Pull request merged**